src/java.base/share/man/java.md

```diff
@@ -3073,9 +3074,9 @@ The following items describe the syntax of `java` argument files:
 -   The argument file size must not exceed MAXINT (2,147,483,647) bytes.
 
 -   The launcher doesn't expand wildcards that are present within an argument
-    file. That means, an asterisk  `*` is passed on as-is to the starting VM.
+    file. That means an asterisk `*` is passed on as-is to the starting VM.
     For example `*.java` stays `*.java` and is not expanded to `Foo.java`,
-    `Bar.java`, etc. like on some command line shell.
+    `Bar.java ...`, as would happen with some command line shells.
```

Comment:
The list was intended to be contiguous - no commas: \'Foo.java Bar.java ...\' as that is how it would be expanded by the shell.

Reply:
Addressed this and the other one in https://github.com/openjdk/jdk/pull/22236/commits/de0ac7c9676e405158c375f469ce1157bdc6f033